accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -27,18 +27,39 @@
 
 import java.util.List;
 
+/**
+ * Service for retrieval and modifications of object accessions.
+ *
+ * @param <MODEL> Type of the objects identified by the accessions
+ * @param <HASH> Type of the hash calculated based on the fields that uniquely identify an accessioned object
+ * @param <ACCESSION> Accession of the object that identify a particular model
```

Comment:
`Type of the accession that identifies an object of a particular model`? To make it consistent with others and more precise.

Reply:
Let's hear from others too

Reply:
It should be Type as Zhicheng mentioned.

Reply:
Agree with @Zhicheng-Liu and @selvaebi .